src/pages/import-cards-popup/import-cards-popup.tsx

```diff
@@ -26,12 +25,23 @@ const cardSeparatorOptions: RadioButtonOptions = {
   '\n': 'new lines',
 };
 
+const enum IMPORT_METHODS {
+  'FILE' = 'FILE',
+  'CLIPBOARD' = 'CLIPBOARD',
+}
+const importMethodOptions: RadioButtonOptions = {
+  [IMPORT_METHODS.FILE]: 'upload file',
+  [IMPORT_METHODS.CLIPBOARD]: 'copy and paste',
+};
+
 export const ImportCardsPopup = ({ showPopup, onClose }: ImportCardsPopupProps) => {
   const [termDefSeparator, setTermDefSeparator] = useState('');
   const [cardSeparator, setCardSeparator] = useState('');
   const [textAreaValue, setTextAreaValue] = useState('');
+  const [importMethod, setImportMethod] = useState('');
+  const [importedFile, setImportedFile] = useState<ImportedFile>();
 
-  const isImportable = termDefSeparator && cardSeparator && textAreaValue.length !== 0;
+  const isImportable = isImportConfigValid();
```

Comment:
for the future, did we want to something more than just disable the `import` button? perhaps some error text could be useful here

Reply:
Yeah, giving more info is a good idea to me. I'll create a ticket for it and mark it as low priority and we can revisit it if we have time